roadmap/index.md

```diff
@@ -4,6 +4,38 @@ layout: default
 
 # Roadmap
 
+## 0.6.0 (August 2018)
+
+### Custom Resource Definitions support for Topic Operator
+
+The Topic Operator should have support for [Custom Resource Definitions (CRDs)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/).
+
+### Add support for encryption to Kafka Connect
+
+Add support for encryption and authentication for Kafka Connect.
+
+### Add support for authentication and authorization
+
+In order to use Strimzi for production workloads, it has to be possible to secure the cluster. This should 
+include:
+
+* Authentication using TLS client authentication and SASL
+* Authorization
+
+## 0.5.0 (July 2018)
+
+### Add support for Kubernetes Affinity and Tolerations
+
+Using Kubernetes Affinity and Tolerations, users can manage scheduling of Kafka, Zookeeper and Kafka Connect pods into nodes.
```

Comment:
Once you've published the blog about this we should link it.

Reply:
Yeah ... I agree ... but we have to release it first :-)